app/src/main/java/com/asfoundation/wallet/wallet_validation/generic/PhoneValidationFragment.kt

```diff
@@ -144,6 +145,22 @@ class PhoneValidationFragment : DaggerFragment(),
 
   override fun setupUI() {
     ccp.registerCarrierNumberEditText(phone_number)
+    ccp.setCustomDialogTextProvider(object : CountryCodePicker.CustomDialogTextProvider {
+      override fun getCCPDialogSearchHintText(language: CountryCodePicker.Language?,
+                                              defaultSearchHintText: String?): String {
+        return defaultSearchHintText ?: ""
+      }
+
+      override fun getCCPDialogTitle(language: CountryCodePicker.Language?,
+                                     defaultTitle: String?): String {
+        return getString(R.string.verification_insert_phone_field_country)
+      }
+
+      override fun getCCPDialogNoResultACK(language: CountryCodePicker.Language?,
+                                           defaultNoResultACK: String?): String {
+        return defaultNoResultACK ?: ""
+      }
+    })
```

Comment:
We can convert to an expression body without specifying the return type.
Do you like this approach?

```
ccp.setCustomDialogTextProvider(object : CountryCodePicker.CustomDialogTextProvider {
      override fun getCCPDialogSearchHintText(language: CountryCodePicker.Language?,
                                              defaultSearchHintText: String?) =
          defaultSearchHintText ?: ""

      override fun getCCPDialogTitle(language: CountryCodePicker.Language?, defaultTitle: String?) =
          getString(R.string.verification_insert_phone_field_country)

      override fun getCCPDialogNoResultACK(language: CountryCodePicker.Language?,
                                           defaultNoResultACK: String?) = defaultNoResultACK ?: ""
    })
```